src/dojos/cd-dojo-details.vue

```diff
@@ -127,6 +127,20 @@
       canAdmin() {
         return !!(this.user && this.user.roles && this.user.roles.indexOf('cdf-admin') > -1);
       },
+      buildTwitterLink() {
+        const twitterData = this.dojoDetails.twitter;
+        if (/^[a-zA-Z0-9_@]{1,15}$/.test(twitterData)) {
+          return `https://twitter.com/${twitterData}`;
+        }
+        return twitterData;
+      },
+      buildFacebookLink() {
+        const facebookData = this.dojoDetails.facebook;
+        if (/^[a-zA-Z0-9.]{1,}$/.test(facebookData)) {
```

Comment:
I don't think so, the regex seems to work when tried in the console with values containing '.'